docs/scos/dev/front-end-development/202212.0/oryx/i18n/localization.md

```diff
@@ -0,0 +1,138 @@
+
+Oryx uses the "current" language to "lookup" the available labels. When the language is `en`, the locales for English are being resolved.
+
+If a language resource can be resolved, the translation key is evaluated against the available translations in the resource, using all "parts" of the key. For example, if you have a translation key that contains multiple parts (e.g. `cart.increase`), you can provide a global translation for just the `increase` part or provide a translation for `cart.increase`. This mechanism allows for a single global translation of `decrease` that might affect multiple components throughout multiple components. This results in a consistent and convenient translation mechanism, while remaining flexible to add very specific translations for some components.
+
+## Auto-conversion of translation keys
+
+If a translation key does not match any of the translations, or if the i18n feature is not installed (which is a default behavior), the translation key is auto-converted to a human readable message.
+
+For example, if you have a token that contains multiple parts (e.g. `cart.increase`), you can provide a global translation for just the `increase` part or provide a translation for `cart.increase`. This mechanism allows for a single global translation of `decrease` that might affect multiple components throughout multiple components. This provides a consistent and convenient translation mechanism, while remaining flexible to add very specific localizations for some components.
+
+The table below shows a few examples to help you understand how the tokens are translated.
+
+| Token                       | Converted label |
+| --------------------------- | --------------- |
+| `cart.increase`             | Increase        |
+| `cart.add-to-cart`          | Add to cart     |
+| `cart.totals.<count>-items` | 5 items         |
+
+This mechanism allows Oryx to **not** distribute any localizations as standard package or as part of the boilerplate. The reason for this is that labels are quit opinionated and might change rapidly over time, which would cause breaking changes.For 90% of the cases the keys are fairly short and provide an OK experience.
+
+_**note:** The [Oryx labs package](https://www.npmjs.com/package/@spryker-oryx/labs) provide some localizations, mainly for demonstration reasons. While it's perfectly fine to use this package for your own demo's or initial project implementation, it is not recommended to rely your production code on this labs package, since the labs package might introduce breaking changes over time (hence there's no major versioning for this package)._

```

Comment:
Maybe add a link to the labs feature description and not add details here? If labs will change in the future - we will have to support this text as well